dyno-jedis/src/main/java/com/netflix/dyno/jedis/DynoDualWriterClient.java

```diff
@@ -174,74 +174,106 @@ public void setRange(int range) {
 
     @Override
     public Long append(final String key, final String value) {
+        return this.d_append(key, value).getResult();
+    }
+
+    @Override
+    public OperationResult<Long> d_append(final String key, final String value) {
```

Comment:
It was not necessary for the fix. I either needed to support it or modify pappy code calling these methods.